### PR TITLE
Credist for crowdin translators

### LINKED ISSUE #3038: Add credits for crowdin translators in app
Add a page somewhere in the app that thanks all the crowdin translators.

I was thinking maybe in the about Copay page add a link below session logs that says "Translators" and the page just lists out the crowdin usernames of everyone who helped.

Then just copy the entire list of translators / proofreaders in crowdin and paste them.

It might be a good way to say thank you. (I think a lot of people who translate the app will evangelize the app too, and saying "I helped translate it, see!" will make the translator more proud to show off the app, and make their friends more likely to try it out.

It will also instill a sense of "I am the xxx translator for Copay" sense of commnuity.

### PR DESCRIPTION
**Pull request by cmgustavo**
fixes #3038

@dabura667 , it 's your idea. Please, let me know if you have more suggestions for this PR.

### CODE REVIEW

File: public/views/translators.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    <li class="line-b p20">Kirvx <span class="right text-gray size-12" translate>French</span></li>
+    <li class="line-b p20">HostFat <span class="right text-gray size-12" translate>Italian</span></li>
+    <li class="line-b p20">koalalorenzo <span class="right text-gray size-12" translate>Italian</span></li>
+    <li class="line-b p20">nicolazza <span class="right text-gray size-12" translate>Italian</span></li>
+    <li class="line-b p20">saschad <span class="right text-gray size-12" translate>German</span></li>
+    <li class="line-b p20">8597fde7 <span class="right text-gray size-12" translate>German</span></li>
+    <li class="line-b p20">pmichelazzo <span class="right text-gray size-12" translate>Portuguese</span></li>
+    <li class="line-b p20">johnblazakis <span class="right text-gray size-12" translate>Greek</span></li>
+    <li class="line-b p20">chek2fire1 <span class="right text-gray size-12" translate>Greek</span></li>
+    <li class="line-b p20">cmgustavo83 <span class="right text-gray size-12" translate>Spanish</span></li>
```
Comment (dabura667) [inline public/views/translators.html]: Do we need the space after everyone's names?

File: public/views/translators.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  </ul>
+  <div class="row column m20t">
+    <div class="size-12 text-gray text-center">
+      <p class="size-12 text-black" translate>All contribution for translation of Copay is welcome. Sign up Crowdin.com and join to Copay project.</p>
```
Comment (dabura667) [inline public/views/translators.html]: > All contributions to Copay's translation are welcome. Sign up at crowdin.com and join the Copay project at <a href="https://crowdin.com/project/copay">https://crowdin.com/project/copay</a>.

What do you think of this instead?

File: public/views/translators.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      <span class="text-bold" translate>There are untranslated languages waiting for translators:</span> Afrikaans, Albanian, Arabic,
```
Comment (dabura667) [inline public/views/translators.html]: I don't think we need to list all languages here.

Maybe

> Don't see your language on Crowdin? Contact the Owner on Crowdin! We'd love to support your language.

File: public/views/translators.html (hunk 1 above)
Comment (cmgustavo) [inline public/views/translators.html]: I did it before to add `right` class :) 

Removed.

File: public/views/translators.html (hunk 2 above)
Comment (cmgustavo) [inline public/views/translators.html]: +1

File: public/views/translators.html (hunk 3 above)
Comment (cmgustavo) [inline public/views/translators.html]: I agree

**Comment by cmgustavo** [comment]
Thanks for reviewing! 
Updated. I also changed the subtitle to `Translation credits` and decreased padding of the list.

**Comment by dabura667** [comment]
LGTM Glad to see it.

Once you merge this, why don't you run grunt translate then run the crowdin_update.js so we can start translation?

**Comment by cmgustavo** [comment]
Sure. I am also waiting for this PR (#3050) before to update .pot file.

**Comment by dabura667** [comment]
OK, looks good. This needs to be updated to templates.pot and then updated to crowdin.

Also, the updateinfo_en.txt file should be updated at least a week or so before next update so that we can have multiple languages of update notes ready for the appstores.

**Pull request merged**